tests/integration/linux_bridge_test.py

```diff
@@ -116,3 +117,21 @@ def test_create_and_remove_linux_bridge_with_two_ports(eth1_up, eth2_up):
         assertlib.assert_state(desired_state)
 
     assertlib.assert_absent(bridge_name)
+
+
+def test_linux_bridge_mac_with_one_iface_uses_default_mac_addr(eth1_up):
+    bridge_name = 'linux-br0'
+    bridge_state = yaml.load(BRIDGE_OPTIONS_YAML, Loader=yaml.SafeLoader)
+    port_state = yaml.load(BRIDGE_PORT_ETH1_YAML, Loader=yaml.SafeLoader)
+    port_name = port_state[LinuxBridge.PORT_SUBTREE][0][LinuxBridge.PORT_NAME]
+    bridge_state.update(port_state)
+
+    with linux_bridge(bridge_name, bridge_state) as desired_state:
+        assertlib.assert_state(desired_state)
```

Comment:
This assert and the one from line 137 are not part of this test. These are assumptions and part of the test setup so there is no need to assert them (have been already checked in previous tests).
I would also move them to a fixture to express these are the test setups:
```
@pytest.fixture
def linux_bridge0_with_single_port(eth1):
    bridge_state = _create_bridge_subtree_config(('eth1',))
    with _linux_bridge(TEST_BRIDGE0, bridge_state) as desired_state:
        yield desired_state
```